rpcserver.go

```diff
@@ -787,7 +787,9 @@ func (r *rpcServer) addDeps(s *server, macService *macaroons.Service,
 		s.sweeper, tower, s.towerClient, s.anchorTowerClient,
 		r.cfg.net.ResolveTCPAddr, genInvoiceFeatures,
 		genAmpInvoiceFeatures, getNodeAnnouncement,
-		s.updateAndBrodcastSelfNode, parseAddr, rpcsLog,
+		s.updateAndBrodcastSelfNode,
+		s.cfg.ProtocolOptions.ExperimentalProtocol.SetFeatureBit(),
```

Comment:
I would squash this commit with the previous one to make it a logical unit of change.